samples/Advanced/ExportSettings/Full.ps1

```diff
@@ -73,6 +73,14 @@ $projectRoot = Split-Path -Parent $scriptsRoot
             account = {$_ -in 'accountid','parentaccountid'} #  only export accountid and parentaccountid fields
         }
     }
+
+	# ExportData is used by export.ps1. Defines configuration data schema file to use for export and the zip file to put the data in.
+    ExportData = [PSCustomObject]@{
+		CrmConnectionParameters = $CrmConnectionParameters
+        SchemaFile ="$projectRoot\temp\export\schema.xml"
```

Comment:
Indentation is not consistent, is sometimes using spaces and other times is using tabs. Please use spaces since the existing code already uses them.